packages/stan/src/test/atom.js

```diff
@@ -4,25 +4,25 @@
 import { createAtomRegistry, createAtom } from '../';
 
 describe( 'atoms', () => {
-	it( 'should allow getting and setting atom values', () => {
+	it( 'should allow getting and setting atom values', async () => {
 		const count = createAtom( 1 );
 		const registry = createAtomRegistry();
-		expect( registry.get( count ) ).toEqual( 1 );
-		registry.set( count, 2 );
-		expect( registry.get( count ) ).toEqual( 2 );
+		expect( await registry.get( count ) ).toEqual( 1 );
+		await registry.set( count, 2 );
+		expect( await registry.get( count ) ).toEqual( 2 );
```

Comment:
Why do we need all the async/wait for the getters for sync atoms and subscriptions?  Note that even if the behavior is sync, JS will make it async and only resolve on the next tick and this breaks the editor (and useSelect) in various ways. It is important that sync selectors stay sync and resolve right away.

Reply:
Good point, I’ll think some more about this on Monday. It seems like Recoil separates atoms and selectors, and only the latyer may be async - maybe that would be a better way to go 🤔